htsinfer/get_read_orientation.py

```diff
@@ -75,8 +75,16 @@ def evaluate(self) -> ResultsOrientation:
         self.mapping.transcripts_file = self.transcripts_file
         self.mapping.tmp_dir = self.tmp_dir
 
-        if not self.mapping.mapped:
+        if not self.mapping.mapped and (
+            self.library_source.file_1.short_name is not None or
+            self.library_source.file_2.short_name is not None
+        ):
             self.mapping.evaluate()
+        else:
+            LOGGER.debug(
+                "Library source is not determined, "
+                "read orientation cannot be inferred by alignment."
+            )
```

Comment:
As above: Please check how other inference failures are logged and use the same logging level. If it's always `debug` then ignore this comment.